sklearn/cluster/hierarchical.py

```diff
@@ -123,6 +124,10 @@ def ward_tree(X, connectivity=None, n_components=None, n_clusters=None,
         limited use, and the 'parents' output should rather be used.
         This option is valid only when specifying a connectivity matrix.
 
+    distance_threshold : int (optional)
+        Stop early the construction of the tree when a given value
+        of distance if obtained.
```

Comment:
IMO, it should be explained here that the user should specify distance_theshold OR n_cluster.